modules/antismash/antismashlitedownloaddatabases/main.nf

```diff
@@ -0,0 +1,34 @@
+process ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES {
+    tag '$database_dir'
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::antismash-lite=6.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/antismash-lite:6.0.1--pyhdfd78af_0' :
+        'quay.io/biocontainers/antismash-lite:6.0.1--pyhdfd78af_0' }"
+
+    input:
+    val(database_dir)
```

Comment:
This confuses me, the database isn't download yet, are you specifying 'where' the database should be downloaded?

I wouldn't do that - if necessary hardcode it into a dir name (e.g. antismash-dbs or something). NXF should deal with the location of internally downloaded files, and the `modules.config` file publishDir options can allow a user tosave this somewhere for re-use.

Alternative if this is necessary, use `prefix` instead